.travis.yml

```diff
@@ -7,143 +7,86 @@ env:
 matrix:
   include:
   - compiler: gcc-4
-    addons:
-      apt:
-        sources:
-        - ubuntu-toolchain-r-test
-        packages:
-        - g++-4.9
+    addons: { apt: { sources: [ ubuntu-toolchain-r-test ] , packages: [ g++-4.9 ] } }
     env:
     - COMPILER=g++-4.9
-    - WARN="-DRAJA_ENABLE_WARNINGS=On"
-  - compiler: gcc-5
-    addons:
-      apt:
-        sources:
-        - ubuntu-toolchain-r-test
-        packages:
-        - g++-5
-    env:
-    - COMPILER=g++-5
-    - WARN="-DRAJA_ENABLE_WARNINGS=On"
+    - CMAKE_EXTRA_FLAGS="-DRAJA_ENABLE_WARNINGS=On"
   - compiler: gcc-6
-    addons:
-      apt:
-        sources:
-        - ubuntu-toolchain-r-test
-        packages:
-        - g++-6
+    addons: { apt: { sources: [ ubuntu-toolchain-r-test ] , packages: [ g++-6 ] } }
     env:
     - COMPILER=g++-6
-    - WARN=""
-    - CMAKE_EXTRA_FLAGS="-DCMAKE_EXE_LINKER_FLAGS='-coverage' -DCMAKE_CXX_FLAGS='-coverage'"
-    - COVERAGE=t
-  - compiler: clang-3.6
-    addons:
-      apt:
-        sources:
-        - ubuntu-toolchain-r-test
-        packages:
-        - g++-5
+    - CMAKE_EXTRA_FLAGS="-DRAJA_ENABLE_WARNINGS=On"
+  - compiler: gcc-7
+    addons: { apt: { sources: [ ubuntu-toolchain-r-test ] , packages: [ g++-7 ] } }
+    env:
+    - COMPILER=g++-7
+    - CMAKE_EXTRA_FLAGS="-DRAJA_ENABLE_WARNINGS=On"
+  - compiler: clang-3.5
+    addons: { apt: { sources: [ ubuntu-toolchain-r-test ] , packages: [ g++-6 ] } }
     env:
-    - COMPILER=$HOME/llvm/clang+llvm-3.6.0-x86_64-linux-gnu/bin/clang++
-    - DOWNLOAD=http://releases.llvm.org/3.6.0/clang+llvm-3.6.0-x86_64-linux-gnu-ubuntu-14.04.tar.xz
-    - WARN=""
-  - compiler: clang-3.7
-    addons:
-      apt:
-        sources:
-        - ubuntu-toolchain-r-test
-        packages:
-        - g++-5
+    - COMPILER=clang++
+    - LLVM_VERSION=3.5.2
+  - compiler: clang-3.9
+    addons: { apt: { sources: [ ubuntu-toolchain-r-test ] , packages: [ g++-6 ] } }
```

Comment:
Switched to a single YAML line for addons since we only install one package from `r-test`